scholia/api.py

```diff
@@ -505,8 +505,8 @@ def entity_to_volume(entity):
     for statement in entity['claims'].get('P478', []):
         volume = statement['mainsnak']['datavalue']['value']
         return volume
-    else:
-        return ''
+
+    return ''
```

Comment:
The documentation to the function says that the function returns an empty string if the field is not set.